Firestore/core/test/firebase/firestore/testutil/testutil.cc

```diff
@@ -25,25 +27,28 @@ std::unique_ptr<model::PatchMutation> PatchMutation(
     const model::ObjectValue::Map& values,
     const std::vector<model::FieldPath>* update_mask) {
```

Comment:
Very optional: can `update_mask` also be a `set`? Perhaps that would allow removing the new constructor of `FieldMask` taking two iterators?